Tasks.9.10.2019/src/test/java/MedianFinderTest.java

```diff
@@ -0,0 +1,45 @@
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+
+public class MedianFinderTest {
+    private MedianFinder mf;
+
+    @Before
+    public void init(){
+        mf = new MedianFinder();
+    }
+
+    @Test
+    public void findMedianTest_even() throws Exception {
```

Comment:
When you have even number of elements, the program theoretically can return either of the two acceptable elements. So you should consider two cases while testing.